src/views/Dashboard.vue

```diff
@@ -48,7 +48,6 @@
                 <StaffNotation
                   v-if="isRoleStaff && !!businessId"
                   addScrollbarOffset="true"
-                  :disabled="!isAllowed(AllowableActions.FILE_STAFF_NOTATION)"
```

Comment:
This check was useless -- it is always true if the current user is staff -- which is already checked on line 49.